tools/ci/website_build.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Note that in general I've been actively trying to move this kind of script from shell into Python, since shell scripts seem to be harder to review an maintain than Python even if they are easier to write in the first place.